app/controllers/verify_controller.rb

```diff
@@ -26,6 +27,7 @@ def app_data
       base_path: idv_app_path,
       completion_url: completion_url,
       initial_values: initial_values,
+      reset_password_url: "#{root_url}/forgot_password",
```

Comment:
I would use the rails router method here instead of building the URL manually
```suggestion
      reset_password_url: forgot_password_url,
```